mailinglist/mailinglist-library/src/main/java/org/silverpeas/components/mailinglist/service/model/dao/MessageDaoImpl.java

```diff
@@ -210,7 +210,7 @@ protected void saveAttachmentFile(Attachment attachment) {
         if (existingFile != null
             && !existingFile.getPath().equals(attachment.getPath())) {
           attachment.setPath(existingFile.getPath());
-          file.delete();
+          deleteFile(file);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Refactor this code to not nest more than 3 if/for/while/switch/try statements. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://integration.silverpeas.org/sonar/coding_rules#rule_key=squid%3AS134)